index.js

```diff
@@ -18,7 +18,7 @@ QUnit.notifications = function(options) {
       var title;
       var _options = {};
 
-      if (window.Notification && QUnit.urlParams.notification === 'true') {
+      if (window.Notification && QUnit.urlParams.notifications) {
```

Comment:
QUnit [official doc](http://api.qunitjs.com/QUnit.config/) says:

> If value is undefined, the option will render as a checkbox. The corresponding URL parameter will be set to "true" when the checkbox is checked, and otherwise will be absent.

In our case, we use:

    window.location = QUnit.url({notification: true});

or

    window.location = QUnit.url({notifications: undefined});

so both are fine. I just though `QUnit.urlParams.notifications` (without ` === 'true'`) is shorter and more readable.